pkg/kubelet/kubelet_getters_windows.go

```diff
@@ -0,0 +1,29 @@
+// +build windows
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	"path/filepath"
+
+	"k8s.io/kubernetes/pkg/kubelet/config"
+)
+
+// getPodResourcesDir returns the full path to the directory containing the pod resources socket
+func (kl *Kubelet) getPodResourcesDir() string {
+	return filepath.Join("//./pipe", config.DefaultKubeletPodResourcesDirName)
```

Comment:
I think kubelet tries to create this directory. Would that be okay?

Reply:
hah, yeah good point. will see if it breaks or not. if it doesn't fail I don't think it will have any effect.